docs/reference.md

```diff
@@ -266,18 +266,76 @@ applied if the marked test method fails or errors.
 Log a summary of the status of the model. This is automatically called before the model
 is cleaned up.
 
-#### `async def track_model(self, alias: str, model_name: Optional[str] = None, cloud_name: Optional[str] = None, create: bool = True, **kwargs,) -> Model`
+#### `async def track_model(self, alias: str, model_name: Optional[str] = None, cloud_name: Optional[str] = None, use_existing: bool = False, keep: Optional[bool] = None, **kwargs,) -> Model`
 
 Indicate to `ops_test` to track a new model which is automatically created in juju or an existing juju model referenced by model_name. 
 This allows support for `ops_test` to track multiple models on various clouds by a unique alias name.
 
+##### Key parameters:
+* `alias`: Required `str` which defines the alias of the model used only by `ops_test`
+* `model_name`: Name of the model referenced by juju -- automatically generated if `None`
+* `cloud_name`: Name of the juju cloud where the model resides -- reuse current cloud if `None`
+* `use_existing`:
+  * `None` (default): `ops_test` will re-use an existing model-name if provided, otherwise False
+  * `False`: `ops_test` creates a new model
+  * `True`: `ops_test` won't create a new model, but will connect to an existing model by `model_name`
+* `keep`: 
+  * `None` (default): inherit boolean value of `use_existing`
+  * `False`: `ops_test` will destroy at the end of testing
+  * `True`: `ops_test` won't destroy at the end of testing
+
+##### Examples:
+
+```python
+# make a new model with any juju name and destroy it when the tests are over
+await ops_test.track_model("alias")
+
+# make a new model with any juju name but don't destroy it when the tests are over
+await ops_test.track_model("alias", keep=True)  
+
+# Invalid, can't reuse an existing model when the model_name isn't provided
+await ops_test.track_model("alias", use_existing=True)
+
```

Comment:
Add one more case:

```
# re-use a model "bob" if it exists, otherwise create a new model named "bob"
await ops_test.track_model("alias", model_name="bob", use_existing=True)
```

Reply:
is there -- just further down in a special section

Reply:
sorry, didn't notice the second section below.  lgtm